gradle/eclipse.gradle

```diff
@@ -2,11 +2,26 @@ import org.gradle.plugins.ide.eclipse.model.AccessRule
 
 apply plugin: 'eclipse'
 
+tasks.eclipse.dependsOn(cleanEclipse)
```

Comment:
To have a consistent build, so that whenever it is executed it does everything from scratch.